handler/user.go

```diff
@@ -0,0 +1,60 @@
+package handler
+
+import (
+	"errors"
+	"net/http"
+	"strconv"
+
+	"github.com/jackc/pgx/v5"
+	"github.com/labstack/echo/v4"
+)
+
+func (h *Handler) GetPartners(c echo.Context) error {
+	idParam := c.Param("id")
```

Comment:
L13-26 seems like a duplicate of L38-51, can we store them inside another function (maybe `getIdAndValidateUser` or some sort).